Makefile

```diff
@@ -0,0 +1,15 @@
+clean: test server
+	@echo "\n\n\nShutting down the server\n"
```

Comment:
you should add all target that runs everything
I think that the kill should be part of the same target
I think you should create a tarter called run_tests or something like that...